server/db/Models/Order.js

```diff
@@ -0,0 +1,44 @@
+const Sequelize = require('sequelize');
+
+const {
+  UUID, UUIDV4, INTEGER, STRING, TEXT, ARRAY,
+} = Sequelize;
+const { db } = require('../db');
+
+const Order = db.define('order', {
+  id: {
+    type: UUID,
+    defaultValue: UUIDV4,
+    primaryKey: true,
+  },
+  quantity: {
+    type: INTEGER,
+    defaultValue: 0,
+    validate: {
+      min: 0,
+    },
+  },
+  movieId: {
+    type: STRING,
+    unique: false,
+  },
+  CartId: {
```

Comment:
Where is the **relationship?** This is not constrained!
If your cart is deleted, this order will be related to carts that don't exist!